tier1/{{cookiecutter.project_slug}}/README.md

```diff
@@ -1,11 +1,12 @@
 ## **{Project Name}**
-
 **{project description}**
 
 ## About the Project
-
 **{project statement}**
 
+### Project Vision
```

Comment:
Let's add this section commented out (recommended/optional) for projects outside of tier2?